_health-care/_js/components/ProgressButton.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import _ from 'lodash';
+
+/**
+ * A component for the continue button to navigate through panels of questions.
+ *
+ * Required props
+ * `onButtonClick`: function that changes the path to the next panel or submit.
+ * `buttonText`: String. Stores the value for the button text.
+ * `buttonClass`: String. Stores the value for the button class(es).
+ */
+
+class ProgressButton extends React.Component {
+  componentWillMount() {
+    this.id = _.uniqueId();
+  }
+
+  render() {
+    return (
+      <div className="row progress-buttons">
+        <div className="small-8 columns">
```

Comment:
One of the reasons it was nice having them stacked is because they have text about the next or previous sections and when they are side by side the text bumps to the next line and it doesn't read as well. I maybe didn't style it the correct way per the mockups though.